package.json

```diff
@@ -31,7 +31,7 @@
     "gettext-extract": "./src/extract-cli.js"
   },
   "dependencies": {
-    "@babel/core": "^7.7.5",
+    "@babel/core": "^7.11.5",
```

Comment:
that could be squashed with the previous commit :)

Reply:
Done